### PR TITLE
Github Action build to temurin

### PR DESCRIPTION
**Pull request by davidjgonzalez**
Testing temurin to align to new Adobe guidance

### CODE REVIEW

**Comment by kwin** [comment]
@davidjgonzalez Is there any reference on Temurin usage with AEM? https://experienceleague.adobe.com/docs/experience-manager-65/deploying/introduction/technical-requirements.html?lang=en#supported-platforms and https://experienceleague.adobe.com/docs/experience-manager-65/assets/adobe-azul-openjdk-license-agreement.pdf?lang=en still list Azul Zulu is replacement of Oracle JDK....

**Comment by davidjgonzalez** [comment]
@kwin  - Adobe is moving off of Azul immediately due to certain political circumstances occurring in the world. Adobe is reverting back to Oracle JDK internally, but i don't believe that is available for our builds. The guidance was "temurin" is the safest bet from the list of supported distro of the setup java GA plugin.

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
         uses: actions/setup-java@v2
         with:
           cache: 'maven'
-          distribution: 'zulu'
+          distribution: 'temurin'
```
Comment (davidjgonzalez) [inline .github/workflows/maven.yml:59]:
```suggestion
          # Adove is moving away from Zule to Oracle JDK (which is not an option) - Temurin is suggested as the "closest thing"
          distribution: 'temurin'
```

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -56,7 +56,8 @@ jobs:
         uses: actions/setup-java@v2
         with:
           cache: 'maven'
-          distribution: 'zulu'
+          # Adove is moving away from Zule to Oracle JDK (which is not an option) - Temurin is suggested as the "closest thing"
```
Comment (davidjgonzalez) [inline .github/workflows/maven.yml:59]:
```suggestion
          # Adobe is moving away from Zule to Oracle JDK (which is not an option) - Temurin is suggested as the "closest thing"
```

File: .github/workflows/maven.yml [hunk 3]
```diff
@@ -56,7 +56,8 @@ jobs:
         uses: actions/setup-java@v2
         with:
           cache: 'maven'
-          distribution: 'zulu'
+          # Adobe is moving away from Zule to Oracle JDK (which is not an option) - Temurin is suggested as the "closest thing"
```
Comment (kwin) [inline .github/workflows/maven.yml:59]:
```suggestion
          # Adobe is moving away from Azul Zulu to Oracle JDK (which is not an option) - Eclipse Temurin is suggested as the "closest thing"
```

**Pull request merged**